xbmc/guilib/GUISpinControl.cpp

```diff
@@ -17,6 +17,12 @@
 #define SPIN_BUTTON_DOWN 1
 #define SPIN_BUTTON_UP   2
 
+namespace
+{
+// Additional space between text and spin buttons
```

Comment:
Not sure if this will show up in doxygen but better to document it using doxy tags rather than an inline comment

```suggestion
/*! Additional space between text and spin buttons  */
```

Reply:
i do not see nothing printed in doxygen regards contents of each unnamed namespaces

Reply:
Disregard then and go for the merge :)